department-of-veterans-affairs/abd-vro
# MCP-2420 Testing MAS incomplete blood pressures

## What was the problem?
While looking at some real data from prodtest I saw that some blood pressure annotation values are partial. Things like “140/-”, “-65”, both of which I saw. We want to make sure that we are 

- not counting these as valid blood pressure readings when assessing sufficiency
- displaying them on the PDF


Associated tickets or Slack threads:

- [MCP-2420](https://amida.atlassian.net/browse/MCP-2420)

## How does this fix it?[^1]


## How to test this PR
- Step 1 build and run app.
- Step 2 put a breakpoint on line 149 of `app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java`
debug `testIncompleteBloodPressures`

pdfToText should show incomplete readings:

`
143/- 9/1/2022 Medical Treatment Record - Government 9/25/2022 823 {2853EB8D-D2BB-4AA1-B8DF-55EB0E7DE1F5}
Facility
-/92 8/21/2022 Medical Treatment Record - Government 9/25/2022 813 {2853EB8D-D2BB-4AA1-B8DF-55EB0E7DE1F5}
Facility
139/77 8/11/2022 Medical Treatment Record - Government 9/25/2022 576, 753, 878 {2853EB8D-D2BB-4AA1-B8DF-55EB0E7DE1F5}
Facility
120/92 7/17/2022 Medical Treatment Record - Government 9/25/2022 83, 86 {2853EB8D-D2BB-4AA1-B8DF-55EB0E7DE1F5}
Facility
167/93 6/20/2022 VAMC record - WASHINGTON VA MEDICAL n.a. n.a. n.a. - data pulled directly from VistA
CENTER
160/100 5/21/2022 Medical Treatment Record - Government 9/25/2022 18, 41 {2853EB8D-D2BB-4AA1-B8DF-55EB0E7DE1F5}`




[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
